fjord/base/email_utils.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.test.client import RequestFactory
+
+
+def render_email(template, context):
+    """Renders a template for email."""
+    req = RequestFactory()
+    req.META = {}
+    resp = render(req, template, context)
+    return resp.content
```

Comment:
I believe you want [render_to_string](https://docs.djangoproject.com/en/1.8/topics/templates/#django.template.loader.render_to_string).